rust/src/filecontainer.rs

```diff
@@ -15,6 +15,8 @@
  * 02110-1301, USA.
  */
 
+//! This module handles file-related operations(open, append, close).
```

Comment:
This should explicitly be file container operations, so its not confused with OS level file operations.